src/test/regress/expected/metadata_sync_from_non_maindb.out

```diff
@@ -121,6 +132,14 @@ select check_database_privileges('grant_role2pc''_user3','metadata_sync_2pc_db',
  (TEMPORARY,t)
 (12 rows)
 
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('grant_role2pc_user4') ORDER BY node_type;
```

Comment:
let's also check

```sql
SELECT node_type, result FROM get_citus_tests_label_provider_labels("grant_role2pc'_user1") ORDER BY node_type;
```

Reply:
Added in the last commit, this requires special handling due to the single quote.